src/test/modules/kitchen-sink/kitchen-sink.model.ts

```diff
@@ -110,6 +110,12 @@ export class KitchenSink extends BaseModel {
   })
   customFieldArrayColumn?: string[];
 
+  @CustomField({
+    api: { type: 'string', nullable: true, sort: false, filter: false, readonly: true },
+    db: { type: 'text', nullable: true, default: 'foobar' }
+  })
+  customTextFieldReadOnly?: string;
```

Comment:
The expectation with this field, since it has `sort: false`, `filter: false` and `readonly: true` is that this will appear on the `KitchenSink` GraphQL model but NOT in any of the filter types or mutation inputs.